netlify.toml

```diff
@@ -2,7 +2,12 @@
 # ##############################################################################
 [build]
   command = "npm run build"
-  publish = ".next"
+  publish = "out"
+
+[build.environment]
+  # For apps that use next export to generate static HTML
+  # set the NETLIFY_NEXT_PLUGIN_SKIP to true.
+  NETLIFY_NEXT_PLUGIN_SKIP="true"
```

Comment:
These need to be set to use static export

Reply:
What? Really? Link to Doxn?

Reply:
Here you go @sherakama
https://nextjs.org/docs/app/building-your-application/deploying/static-exports

It would also show me an error in on the Netlify build page to do these two things

Reply:
TBH I'm not 100% sure I want to go with this - but at least I finally get it to work once 😂 All the combination of things that blocked this from working before.....